.github/workflows/php-hook-documentation.yml

```
@@ -0,0 +1,48 @@
+name: PHP Hook Documentation Generator
+
+on:
+  push:
+    branches:
+      - "release/**"
+    paths:
+      - changelog.txt # Run the workflow only after the last commit of Woo Release. When it updates the changelog.
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  HookDocumentation:
+    name: Hook Documentation Generator
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+        with:
+          # Checks out a branch instead of a commit in detached HEAD state
+          ref: ${{ github.head_ref }}
+
+        # This generates the documentation string. The `id` property is used to reference the output in the next step.
+      - name: Generate hook documentation
+        id: generate-hook-docs
+        uses: woocommerce/grow/hook-documentation@actions-v1
+        with:
+          source-directories: includes/ woocommerce-google-analytics-integration.php
```

> However, when I tested it, I realized that Hooks Documentation Generator is not really working for this repo.

I pushed 5a822245229a55c0f3309ad4be97ff7116983949 to #294 and the document was updated successfully. See: https://github.com/woocommerce/woocommerce-google-analytics-integration/pull/294/files/e2621507048da79b2285b45417da91a28c36b2c2..1764a5bdd881e577be99fe52ba9daa2cd7e4b37a

l think this should fix the issue:

```suggestion
          source-directories: includes/,woocommerce-google-analytics-integration.php
```

---

Thanks @eason9487 , still in low mode after the fever so I missed that!